.gitignore

```diff
@@ -16,6 +16,7 @@ config.*
 !cmake/config.in
 *Debug/
 *Release/
+*Release (options.h)/
```

Comment:
Why only for release? Where is the README to go along with these new VS 2022 projects? Would be nice to explain the various configuration options and where the user_settings.h is coming from.

Reply:
> Why only for release? 

The initial settings for only release build was as related to https://github.com/wolfSSL/wolfcrypt_nodejs/pull/8 where debugging was unlikely.

It was also an exploratory PR, to see if there was interest in an `options.h` build for Windows. 

But you are right; there should be a debug option. Potentially also `options.h` configs for 32bit debug and release, too.

> Where is the README to go along with these new VS 2022 projects? 

I neglected that, as I was documenting in an upcoming blog that I would have referred to. 

I'll add some README info, as well.

> Would be nice to explain the various configuration options and where the user_settings.h is coming from.

That's also in the upcoming blog, but you ae correct it should also be in the README.

Reply:
Seems like documentation needs to be added similar to PR description?